MetaMask/metamask-extension
# chore: bump asset controllers to 39 + polling API

## **Description**

Bumps the asset controllers to 39.  In this version, the polling API has changed from `startPollingByNetworkClientId` to a more flexible `startPolling` that can accept any input.   The `usePolling` hook is modified to accommodate this.


[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/28025?quickstart=1)

## **Related issues**


## **Manual testing steps**

Should be no noticeable changes.  Currency rates should continue to be fetched.  Try switching between chains with different native currencies like mainnet, polygon, bnb and verify the native token has fiat prices.

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
